app/src/lib/design-system-v2/components/RQButton/RQButton.scss

```diff
@@ -198,3 +197,18 @@
     }
   }
 }
+
+.rq-custom-btn.ant-btn.source-filter-icon-container {
+  color: var(--requestly-color-primary-400);
+}
+.rq-custom-btn.ant-btn.source-filter-icon-container .badge-default {
+  display: none;
+}
+.rq-custom-btn.ant-btn.source-filter-icon-container .badge-active {
+  color: var(--requestly-color-text-default);
+  background-color: var(--requestly-color-primary-400);
+  border-radius: 4px;
+  font-size: 12px;
+  padding: 2px 6px;
+  line-height: 16px;
+}
```

Comment:
Move this styles [here](app/src/components/features/rules/RulePairs/Pairs/Rows/RowsMarkup/RequestSourceRow/RequestSourceRow.css)